### PR TITLE
Fix Keep or Sweep for Nextcloud 14, fix #17

### LINKED ISSUE #17: Layout is broken on Nextcloud 14
https://github.com/nextcloud/server/pull/9982

### PR DESCRIPTION
**Pull request by jancborchardt**
Fix the layout according to https://github.com/nextcloud/server/pull/9982, basically only using #app-content.

Please review @xMartin @camilasan @nickvergessen :pray: also upped the version cause I’ll make a release with that, and with it we will only support Nextcloud 14 and later.

Before & after:
screenshot from 2018-08-14 12-44-07
screenshot from 2018-08-14 12-42-56

### CODE REVIEW

**Comment by jancborchardt** [comment]
Anyone up for a quick double-check? Maybe @dosch @thatscoteng @Sanookmakmak @aproposnix @happyreacer @borekon?

Any review is appreciated! :heart_eyes: :tada:

**Comment by dosch** [comment]
screen shot 2018-08-21 at 09 26 19
I mainly miss the keep button. (it is there, I can click it, it just doesn't show...)

File: appinfo/info.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@ Currently this works for Files, and we have [plans to integrate many more source
     </documentation>
 
     <dependencies>
-        <nextcloud min-version="12" max-version="14"/>
+        <nextcloud min-version="14" max-version="15"/>
```
Comment (nickvergessen) [inline appinfo/info.xml]: I wouldnt claim that is works with 15 now...

Comment (jancborchardt) [reply appinfo/info.xml]: Yeah, but the point is that directly after 14 release the main version for devs will be 15, and nothing initially changes. :)

**Comment by aproposnix** [comment]
@jancborchardt sorry for the delay in response. To be honest, I had to remove the app because I have some issues with it.  I've gone ahead and reinstalled on the latest 14rc release and the alignment is still broken for me. 

Here's a screenshot: http://pix.toile-libre.org/upload/original/1535089951.png

**Comment by jancborchardt** [comment]
@aproposnix this pull request should fix exactly that alignment issue. :)

It would be great if you could open separate issues for the other problems you encounter with the app too – any help is appreciated. :)

**Comment by aproposnix** [comment]
@jancborchardt Sorry. Didn't realize I was reading a pull request ticket. :) 

With regards to your other point, it's a usability issue I reported on issue #9 
As it is now, I have to be very careful cleaning as I have on multiple occasions, accidentally deleted files not meant for deletion.  I hope someone could tackle this request as I think it will add a lot of value for people needing to clean out auto-synced image files.

**Pull request merged**